subjects/DPxAI/robots-harmony/README.md

```diff
@@ -0,0 +1,394 @@
+## Robots Harmony
+
+> AI Synergy Mode
+
+## Context
+
+Welcome to your final milestone! After each of you has brought your unique robot to life, it's time to reunite them together.
+
+Now, all those incredible robots need to be displayed in perfect harmony in our interactive and visually stunning Robots `Gallery`.
+
+As a team, your task is to combine your individual creations into a cohesive display.
+
+This gallery won't just be functional, it will be a fun and visually appealing experience that highlights the creativity and collaboration behind your robots.
+
+> You'll be working in this mission as a team, so effective communication and mutual support are key to bringing your robots together.
+
+> Separate tasks equally, it will make the results better!
+
+> Remember what you learned in asking AI for the precise explanation of concepts, while your team apply those concepts and do the actual fun work!
+
+Go ahead, and let the world see the amazing robots you've created together!
+
+## Setups:
+
+**1- Code Editor:**
+
+First , remember that you will not be using the platform's code editor. Instead, you can use this online [tool](https://jsfiddle.net/) to create and modify your code.
+
+> Feel free to play with the settings and familiarize yourself with the environment!
+
+**2- How to submit your code:**
+
+------->Louis
+
+**3- Expected files:**
+
+You need to submit the following files:
+
+- `robots-harmony.html`
+- `robots-harmony.css`
+- `robots-harmony.js`
+
+## Instructions
+
+Theses tasks are representing the mandatory part for this raid to be passed successfully. Bonus part is optional but highly encouraged for more fun!
+
+### Task 1: Set Up the Gallery Structure
+
+#### 1- Create a New HTML File:
+
+Start by creating a new HTML file called `robot-harmony.html`.
+
+Set up the basic structure of the HTML document with a `<head>` and `<body>` inside an `<html>` tag.
+
+#### 2- Create a Container for the Robots:
+
+Inside the `<body>`, create a div element with the ID gallery.
```

Comment:
ID gallery could be in a format more visible ? like body in this sentence

Reply:
Fixed.